package-lock.json

```diff
@@ -9,232 +9,232 @@
 			"resolved": "https://registry.npmjs.org/@gov.au/accordion/-/accordion-5.0.2.tgz",
 			"integrity": "sha512-+vUdlxkUa90uRTlA3wRgmeTp7oZcjx6BbnOLRvTc41NB6XgB9TyevOFGsg+2G6YbGQV4G9CRfwugXoEYP+quNQ==",
 			"requires": {
-				"@gov.au/animate": "^1.0.0",
-				"@gov.au/core": "^3.0.0",
-				"@gov.au/pancake": "~1",
-				"@gov.au/pancake-js": "~1",
-				"@gov.au/pancake-json": "~1",
-				"@gov.au/pancake-react": "~1",
-				"@gov.au/pancake-sass": "~2"
+				"@gov.au/animate": "1.0.8",
+				"@gov.au/core": "3.0.3",
+				"@gov.au/pancake": "1.2.4",
+				"@gov.au/pancake-js": "1.0.13",
+				"@gov.au/pancake-json": "1.0.7",
+				"@gov.au/pancake-react": "1.1.4",
+				"@gov.au/pancake-sass": "2.1.0"
```

Comment:
Can we recreate this file, seeing some weird stuff - it might be all good though.